modules/AlphaWalletFoundation/AlphaWalletFoundation/Transfer/Types/SendTransactionError.swift

```diff
@@ -2,15 +2,25 @@
 
 import Foundation
 
-public enum SendTransactionNotRetryableError: Error {
-    case insufficientFunds(message: String)
-    case nonceTooLow(message: String)
-    case gasPriceTooLow(message: String)
-    case gasLimitTooLow(message: String)
-    case gasLimitTooHigh(message: String)
-    case possibleChainIdMismatch(message: String)
-    case executionReverted(message: String)
-    case unknown(code: Int, message: String)
+public struct SendTransactionNotRetryableError: Error {
+    public enum ErrorType {
```

Comment:
Why is it neccessary to wrap it with a separate `ErrorType` enum?

Reply:
enum cant keep variables and as some of cases want't rpcserver i wrapped it into ErrorType with being able to keep server as well for error

Reply:
Oh right. But `server` isn't used :P